bert/src/bert_layers.py

```diff
@@ -0,0 +1,859 @@
+# coding=utf-8
```

Comment:
It would be helpful somewhere (e.g. in a README) to explain in detail what each architecture choice such as GLU and ALiBi does, and how it is implemented in the code. Not blocking.

Reply:
Added a header and pointed out which modules handle which new choices throughout.